tests/acceptance/test_mender-connect.py

```diff
@@ -193,14 +202,22 @@ def test_mender_connect_auth_changes(
         """Test that mender-connect can re-establish the connection on D-Bus signals"""
 
         try:
-            dbus_set_token_and_url(connection, "token1", "http://localhost:5000")
-
             # start the mender-connect service
-            startup_time = qemu_system_time(connection)
             connection.run(
                 "systemctl --job-mode=ignore-dependencies start mender-connect"
             )
 
+            startup_time = qemu_system_time(connection)
+            # wait until the connection happens
+            wait_for_string_in_log(
+                connection, startup_time, 30, "Started Mender Connect service"
+            )
+
+            signal_time = qemu_system_time(connection)
```

Comment:
Shouldn't we use `signal_time` below (line 224) when looking for "Connection established..." in the log?

Reply:
Oh sorry my bad, fixed it now :+1: